segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/ZooKeeperServiceRunner.java

```diff
@@ -55,6 +80,34 @@ public void initialize() throws IOException {
         if (this.tmpDir.compareAndSet(null, IOUtils.createTempDir("zookeeper", "inproc"))) {
             this.tmpDir.get().deleteOnExit();
         }
+        if (secureZK) {
+            //-Dzookeeper.serverCnxnFactory=org.apache.zookeeper.server.NettyServerCnxnFactory
+            //-Dzookeeper.ssl.keyStore.location=/root/zookeeper/ssl/testKeyStore.jks
+            //-Dzookeeper.ssl.keyStore.password=testpass
+            //-Dzookeeper.ssl.trustStore.location=/root/zookeeper/ssl/testTrustStore.jks
+            //-Dzookeeper.ssl.trustStore.password=testpass
+            System.setProperty("zookeeper.serverCnxnFactory", "org.apache.zookeeper.server.NettyServerCnxnFactory");
+
+            System.setProperty("zookeeper.ssl.keyStore.location", this.keyStore);
+            //TODO: Read these from the config/parameter files..
```

Comment:
No `TODO` without an associated issue, please.

Reply:
This is already fixed. Removing `TODO`.